### ISSUE TITLE
analysis idea: are there any data that look mis-labelled?

### ISSUE THREAD

**Issue by taylorreiter** [dib-lab/2022-sra-gather#1]
I was just messing around with the outputs to figure out what the file endings meant, and I peeked at ERR1897397:
```
overlap     p_query p_match avg_abund
---------   ------- ------- ---------
100.0 kbp     74.1%  100.0%      20.0    GCA_903905285.1 uncultured Alphaprote...

found 1 matches total;
the recovered matches hit 74.1% of the abundance-weighted query
```

Methinks this looks not like a metagenome

Which gave me the idea, that it might be fun to try and see if things are mislabelled:
+ genome as a metagenome
+ 16s as a metagenome
+ metatranscriptome as a metagenome (this one might be hard)
+ biome as the wrong biome (also might be hard, but random forests would probably pull these samples out)
+ human-associated not labelled as human associated (I would think there would be matches to the human sig or something, although lab contamination could also cause this)

probably other things too :)

**Comment by ctb** [comment]
yep - I also wonder how many "isolates" are not, in fact, isolate... but that's out of scope for an SRA metagenome search I guess 😆

**Comment by ctb** [comment]
I definitely like the idea of a fairly extensive and iteratively built "triage" pipeline that successively weeds out the obvious problems - host contamination included.
